mobile/library/cc/engine_builder.h

```diff
@@ -152,6 +152,8 @@ class EngineBuilder {
   EngineBuilder& enableSocketTagging(bool socket_tagging_on);
 #ifdef ENVOY_ENABLE_QUIC
   EngineBuilder& enableHttp3(bool http3_on);
+  EngineBuilder& setHttp3ConnectionOptions(std::string options);
```

Comment:
I wonder if this should be a span<string> or if we prefer to use string to be simpler?

Reply:
ditto.
I believe having a string is simpler and faster